### ISSUE TITLE
Feature Request - Clickable rows with modal edit

### ISSUE THREAD

**Issue by Nuranto** [leantony/laravel-grid#63]
I think the title is explicit enough, tell me if you want precision.

**Comment by leantony** [comment]
Yes, I understand. The problem is, the `linkable` class is added on the <tr> element. And since this element spans the entire row, the buttons on the row get affected, so the navigation just gets handled and the modal is not displayed.
See https://github.com/leantony/laravel-grid/blob/4e28cf73fce58c9a5bb18868b647ace599600914/src/resources/views/grid/grid.blade.php#L116

I can try wrap the row portion upto the buttons in a div, and add the `linkable` class to it as a solution. Will try this
